contrib/azure-setup.sh

```diff
@@ -137,6 +137,7 @@ createResourceGroup() {
 
 createStorageAccount() {
 	echo "==> Creating storage account"
+    azure provider register Microsoft.Storage
```

Comment:
NIT: indentation.

I am not sure what happens if a resource is already registered.  Does XPlat return zero or non-zero status?  I would eat the error, and continue on.